tests/regex_test.py

```diff
@@ -251,7 +251,7 @@ def test_regex_peel(self):
         assert ["", "", "1f2g"] == d_right.to_list()
         assert ["", "", "__f____g"] == u_right.to_list()
 
-    def test_regex_flatten(self):
+    def test_regex_split(self):
```

Comment:
You've renamed two different functions to "regex_split."  Is that intentional?

Reply:
I renamed the second one to t`est_regex_regex_split`, since is is a `regex` test for `regex_split`.